airbyte-integrations/connectors/destination-iceberg/src/main/java/io/airbyte/integrations/destination/iceberg/IcebergConstants.java

```diff
@@ -27,6 +27,9 @@ public class IcebergConstants {
   public static final String JDBC_PASSWORD_CONFIG_KEY = "password";
   public static final String JDBC_SSL_CONFIG_KEY = "ssl";
   public static final String JDBC_CATALOG_SCHEMA_CONFIG_KEY = "catalog_schema";
+  public static final String REST_CATALOG_URI_CONFIG_KEY = "rest_uri";
+  public static final String REST_CATALOG_CREDENTIAL_CONFIG_KEY = "rest_credential";
```

Comment:
REST catalog uses either a token or a credential to interact with the REST server: https://github.com/apache/iceberg/blob/apache-iceberg-1.3.0/core/src/main/java/org/apache/iceberg/rest/auth/OAuth2Properties.java#L24-L28